tests/python_functional/functional_tests/destination_drivers/snmp_destination/general/test_snmp_destination_reload_statistics.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+#############################################################################
+# Copyright (c) 2020 One Identity
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 as published
+# by the Free Software Foundation, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+#
+# As an additional exemption you are allowed to compile & link against the
+# OpenSSL libraries as published by the OpenSSL project. See the file
+# COPYING for details.
+#
+#############################################################################
+import pytest
+
+from src.helpers.snmptrapd.conftest import *  # noqa:F403, F401
+
+
+@pytest.mark.snmp
+def test_snmp_dest_reload_restart_stat(config, syslog_ng, syslog_ng_ctl, snmptrapd, snmp_test_params):
+    input_message_counter = 1
+    # example_msg_generator source generates a new message on start
+    output_message_counter = 1
```

Comment:
this variable initialization does not needed here

Reply:
done